nilearn/decoding/tests/test_searchlight.py

```diff
@@ -27,11 +29,13 @@ def test_searchlight():
     data[2, 2, 2][cond.astype(np.bool)] = 2
     data_img = nibabel.Nifti1Image(data, np.eye(4))
 
-
     # Define cross validation
     from sklearn.cross_validation import check_cv
     # avoid using KFold for compatibility with sklearn 0.10-0.13
-    cv = check_cv(4, cond)
+    with warnings.catch_warnings():
+        #  check_cv will return indices instead of boolean masks from 0.17
+        warnings.simplefilter('ignore', DeprecationWarning)
+        cv = check_cv(4, cond)
```

Comment:
It may be just me but I'm a bit worried about these catchs. I have the impression that we're putting a very high standard on tests that may frighten potential contributors. @lesteve sees no problem so this is a matter of taste.